compiler/src/dotty/tools/dotc/sbt/ExtractAPI.scala

```diff
@@ -651,7 +712,7 @@ private class ExtractAPICollector(using Context) extends ThunkHolder {
       tp match
         case ConstantType(c) =>
           h = constantHash(c, h)
-        case TypeBounds(lo, hi) =>
+        case TypeBounds(lo, hi) => // TODO when does this happen?
```

Comment:
So does this actually ever happen?

Reply:
I don't think I can manually find a situation, I will just leave it out and we will see in CI